cypress/integration/v2/features/dynamic-content/site-data.spec.js

```diff
@@ -0,0 +1,72 @@
+
+/**
+ * External dependencies
+ */
+import { registerTests } from '~stackable-e2e/helpers'
+describe( 'Dynamic Content - Site', registerTests( [
+	  adjustSiteData, matchSiteData,
+] ) )
+
+const fields = {
+	 'Site Title': 'title',
+	 'Site Tagline': 'tagline',
+	 'Site URL': 'url',
+}
+
+function adjustSiteData() {
+	 it( 'should adjust the site data', () => {
+		 cy.loginAdmin()
+		 //find and click customize
+		 cy
+			 .contains( 'Customize Your Site' )
+			 .click( { force: true } )
+		 //find 'site identity' and edit fields for site title, tagline
+		 cy
+			 .contains( 'Site Identity' )
+			 .click( { force: true } )
+
+		 cy
+			 .get( '[id$=-input-blogname]' ).clear( { force: true } )
+			 .type( 'Test Site Title', { force: true } )
+			 .get( '[id$=-input-blogdescription]' ).clear( { force: true } )
+			 .type( 'Test Site Tagline', { force: true } )
```

Comment:
No need to add this here. We already have a command in master branch to adjust site title & site tagline. See https://github.com/gambitph/Stackable-e2e-tests/tree/master/cypress/support/wordpress/commands
`editSiteTitle` & `editSiteTagline`
check the readme file for the documentation